Assets/PatchKit Patcher/Scripts/UI/AppChangelogText.cs

```diff
@@ -10,7 +10,7 @@ namespace PatchKit.Unity.UI
 {
     public class AppChangelogText : AppCompontent
     {
-        [Multiline] public string Format = "<b>{label}</b>  Published at: {publishdate}\n{changelog}\n\n";
+        [Multiline] public string Format = string.Format("<b>{label}</b>  {0} {publishdate}\n{changelog}\n\n",PatcherLanguages.GetTraduction("published_at"));
```

Comment:
To make it work you need to make similar things as in `LicenseDialog` (removing the field from the inspector, and using the translation directly in the code).

Reply:
if I understood correctly it was done